java assignments/assignments/assignment3/PingHost.java

```diff
@@ -0,0 +1,47 @@
+package com.assignments.assignment3;
+
+import java.net.*;
+import java.util.*;
+public class PingHost {
+    public static void main(String args[])
+    {
+        long start,end;
+        boolean a;
+        InetAddress address1;
+        int n,i;
+        Scanner sc=new Scanner(System.in);
+        System.out.println("enter the no of hosts you want to ping:");
+        n=Integer.parseInt(sc.nextLine());
+        String site;
+        long time[]=new long[n];
+        try {
+            i=0;
+            while(i<n)
+            {
```

Comment:
better to use seperate method for caluclating average as everything in a single method is not a good way.